analysis/paladinholy/src/modules/shadowlands/tier/Tier28FourSet.tsx

```diff
@@ -0,0 +1,92 @@
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Spell from 'common/SPELLS/Spell';
+import SpellLink from 'interface/SpellLink';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { HealEvent, CastEvent } from 'parser/core/Events';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import HealingDone from 'parser/shared/modules/throughput/HealingDone';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+const CDR = 1000;
+
+class Tier28FourSet extends Analyzer {
+  static dependencies = {
+    spellUsable: SpellUsable,
+    healingDone: HealingDone,
+  };
+  cooldownReductionUsed: number = 0;
+  cooldownReductionWasted: number = 0;
+  WingsCasts: number = 0;
+  spellToReduce: Spell = SPELLS.AVENGING_WRATH;
+  healingBoost: number = 0;
+  healing: number = 0;
```

Comment:
Don't think you need healing/healingBoost/conduitRank etc. here, make sure to remove stuff when you're done with it :)